shm/src/Store.cpp

```diff
@@ -16,6 +16,13 @@ Store::Response Store::buy(Cargo* cargo, size_t amount, Player* player) {
     return Response::done;   
 }
 
+Store::Response Store::sell(Cargo* cargo, size_t amount, Player* player) {
+    if (cargo->getAmount() + amount > Store::STORE_CAPACITY) {
```

Comment:
No member named 'STORE_CAPACITY' in 'Store'; did you mean simply 'STORE_CAPACITY'?